src/help.ts

```diff
@@ -41,6 +41,7 @@ Options:
   --since             - for the bump command: allows to specify the range of change files used to bump packages by using git refs (branch name, commit SHA, etc);
                         for the publish command: bumps and publishes packages based on the specified range of the change files.
   --keep-change-files - for the bump and publish commands: when specified, both bump and publish commands do not delete the change files on the disk.
+                        for the check command: verifies that no change files were deleted between head and target branch.
```

Comment:
It feels a bit strange to have this same CLI option do two very different things. Also, should this be opt out instead of opt-in?

Reply:
I'm open to adding a new flag. Do you think `--no-change-files-deleted` suits this purpose better? I think this flag should stay opt-in, otherwise this change won't be backwards compatible with the existing behavior of beachball.

Reply:
Yeah, I like no change files deleted!

Reply:
I've switched to `disallow-deleted-change-files` because `yargs` logic for the negation of booleans prefixed with `no-*` complicates everything.